manifests/templates/otel-collector.yaml

```diff
@@ -40,9 +41,205 @@ data:
       extensions: [health_check]
       pipelines:
         metrics:
+          receivers: [opencensus]
+          processors: [batch]
+          exporters: [logging]
+---
+apiVersion: v1
```

Comment:
Why is this ConfigMap here if it's already in a constant in `otel_collectore.go`?

Isn't it supposed to be optional, only added for GCP clusters with workload identity enabled?

Reply:
Ignore this file too :]